### PR TITLE
Write safer negative report

### LINKED ISSUE #58: Change the message if no leak found
The wording of messages _“Hooray, everything is safe!”_ and _“Your password is safe!”_ implies these passwords are safe. Actually they only have not been reported as unsafe _until now_.

How about: _“Your password is probably safe.”_ or _“This password has not been reported unsafe until now.”_ or let's find a better message in the discussion.

### PR DESCRIPTION
**Pull request by eumiro**
Tell the user their password has not been reported leaked and not their password is safe.

Fixes #58

### CODE REVIEW

**Comment by jugmac00** [comment]
Thank you for the PR!

Would you please add a change log entry?

Maybe in the form of:
- text text text (eumiro)

I saw this in another repository and I like it, as it highlights external contributors!

**Comment by eumiro** [comment]
Voilà.

File: CHANGES.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ added
 - add the `bandit` security checker
 - add support for Python 3.9
 - add type annotations
+- tell the truth about safe password (@eumiro)
```
Comment (jugmac00) [inline CHANGES.rst:12]:
```suggestion
- improve message when there are no known leaks for a password(@eumiro)
```

**Comment by jugmac00** [comment]
Thanks!

**Pull request merged**